### PR TITLE
Moved wiki install guides into installation.md directly

### LINKED ISSUE #5868: Remove all install guides from wiki and move to source control
## Description of your problem
People can edit wiki silently which is an issue for install guides
https://twitter.com/pymc_devs/status/1534472124072591362

Moving to source control means they can't be edited without visibility and approval

### PR DESCRIPTION
**Pull request by fonnesbeck**
Closes #5868

### CODE REVIEW

**Comment by twiecki** [comment]
We can just have one set of instructions across all OS.

**Review by OriolAbril — CHANGES_REQUESTED** [review]
As part of the docs this will now be versioned and therefore should have no pymc3 content

**Comment by fonnesbeck** [comment]
Except the PyMC3 install instructions aren't yet in a version, are they?

**Comment by OriolAbril** [comment]
> Except the PyMC3 install instructions aren't yet in a version, are they?

3.11 docs points to the wiki for install instructions: https://www.pymc.io/projects/docs/en/v3.11.5/ (there is one clickable card per OS in the installation section). Killing the wiki is the issue with 3.x install instructions, I don't think adding them on 4.x docs will fix anything on that end.

File: docs/source/installation.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+```console
+conda create -c conda-forge -n pymc_env pymc
+conda activate pymc_env
+```
+If you like, replace the name `pymc_env` with whatever environment name you prefer.
+
+
+## JAX sampling
+
+If you wish to enable sampling using the JAX backend via NumPyro (experimental), the following should also be installed:
```
Comment (OriolAbril) [inline docs/source/installation.md]: is only numpyro experimental but not blackjax? If so I'd change the order and put blackjax first

File: docs/source/installation.md [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+
+```console
+pip install jax jaxlib numpyro
```
Comment (OriolAbril) [inline docs/source/installation.md]: I find it a bit confusing that jax and jaxlib are included here but not for blackjax. Both numpyro and blackjax depend on jax and jaxlib and will install both packages when installing them. I'd suggest doing the same in both cases (either with or without jax+jaxlib, that part I don't really care)

File: docs/source/installation.md (hunk 1 above)
Comment (twiecki) [inline docs/source/installation.md:17]:
```suggestion
If you wish to enable sampling using the JAX backend via NumPyro or BlackJAX, the following should also be installed:
```

Comment (twiecki) [reply docs/source/installation.md]: I don't think it's that experimental anymore.

Comment (fonnesbeck) [reply docs/source/installation.md]: I assumed it was experimental until it was integrated into `pm.sample` ... happy to drop it though,

File: docs/source/installation.md (hunk 2 above)
Comment (fonnesbeck) [reply docs/source/installation.md]: Actually, its the reverse! Numpyro will install jax and jaxlib by default, but blackjax will not. I will switch them.

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
```

 Note that JAX is not directly supported on Windows systems at the moment.
-
Error: The process '/opt/hostedtoolcache/Python/3.10.4/x64/bin/pre-commit' failed with exit code 1

```

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**